btcec/schnorr/musig2/keys.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2013-2016 The btcsuite developers
+// Copyright (c) 2015-2021 The Decred developers
```

Comment:
Is there actual code being copied from Decred? Or could this just be `2013-2022 The btcsuite developers`? Same in all other files.